<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главный экран</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>608</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deauthorizationButton">
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 5px 14px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>258</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
         <property name="text">
          <string>Списки предметов</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>258</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
           </property>
           <property name="text">
            <string>Название предмета</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="creatingSubject">
           <property name="styleSheet">
            <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
           </property>
           <property name="text">
            <string>Код предмета</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="creatingCodeSubject_2">
           <property name="styleSheet">
            <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
         <property name="text">
          <string>Преподаватель</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="TeacherChoose">
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="creatingSubjectSubmit">
       <property name="styleSheet">
        <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
       </property>
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
